server/routes/app.js

```diff
@@ -20,6 +20,7 @@ const template = ({client, config, version}) => `
       <div id="app"></div>
       <script>window.__client__ = ${JSON.stringify(client)}</script>
       <script>window.__config__ = ${JSON.stringify(config)}</script>
+      <script>window.__api__ = ${JSON.stringify(api)}</script>
```

Comment:
If I understand correctly, `api` (and `window.__api__`) will contain only API errors, because the API data itself is inside `config` (and `window.__config__`). So shouldn't this be named `error` (and window.__error__) or similar? Or do you envision more data to be added to this object?

Reply:
Done